app/importers/trade_event/trade_event_data.rb

```diff
@@ -0,0 +1,9 @@
+module TradeEvent
+  class TradeEventData
```

Comment:
`BaseData`?

I often wanted to rename the importers from *Data to *Importer. Didn't ever get around to it though.